src/Alchemy/BinaryDriver/Exception/ExecutionFailureException.php

```diff
@@ -11,6 +11,29 @@
 
 namespace Alchemy\BinaryDriver\Exception;
 
+use Throwable;
```

Comment:
where is this used?

Reply:
TLDR: It's safe to remove it.

Oh my bad. That comes from an autogenerated stub from my IDE. It was for type-hinting the last parameter from the constructor, but since Throwable is not recognized on PHP 5, I removed the type-hint but I forget removing the namespace import.

Reply:
Already removed it and updated the PR